src/vscode-bicep/package.json

```diff
@@ -36,6 +36,7 @@
   "homepage": "https://github.com/Azure/bicep/blob/main/README.md",
   "activationEvents": [
     "onLanguage:bicep",
+    "onLanguage:bicep-params",
```

Comment:
>onLanguage [](http://example.com/codeflow?start=5&length=10)

Needed these changes so I can manually test params. I will drop the commit before merging.